content/en/docs/usecases/configuration.md

```diff
@@ -14,7 +14,7 @@ data templating languages.
 CUE basic operation merges configurations in a way that the outcome is
 always the same regardless of the order in which it is carried out
 (it is associative, commutative and idempotent).
-This property enables many of the its favorable properties, as discussed below.
+This property enables many of its favorable properties, as discussed below.
```

Comment:
> This property enables many of its favorable properties, as discussed below.

I actually think this sentence was intended as something like:

> This property is the foundation for many other favorable properties, as discussed below.

Reply:
That seems like a good suggestion, but I am still at the beginning of learning Cue, so I will defer to the project owners to make a call on it.

Reply:
If you're happy making this change as part of this PR please do. Otherwise I'm happy to make it in a follow up. Thanks again

Reply:
Done.